docs/src/main/tut/usage.md

```diff
@@ -253,15 +259,8 @@ You can also get a stream of changes with `changes(p)` where `p` is some `Patter
 
 If you're running *Knobs* from within an application that is hosted on AWS, you're in luck! *Knobs* comes with automatic support for learning about its surrounding environment and can provide a range of useful configuration settings. For example:
 
-```
-import knobs._
-
-val c1: Task[Config] =
-  knobs.loadImmutable(Required(FileResource(someFile)) :: Nil)
-
-val cfg: Task[Config] =
-  c1.flatMap(AWS.config)
-
+```tut
```

Comment:
Oh yeah, thanks - this has actually always been wrong. It should be:

```
val cfg = for {
  a <- c1
  b <- AWS.config
} yield a ++ b
```
The idea was to show how to just include the AWS settings as a part of your configuration.